modules/automod/index.ts

```diff
@@ -59,6 +59,9 @@ defineEvent.pre("interactionCreate", async (interaction) => {
 	return true;
 });
 defineEvent.pre("messageCreate", async (message) => {
+	(await config.guild.members.fetch(message.author.id)).roles
+		.add("1021061241583714304")
```

Comment:
excuse me?

Reply:
oh shoot

Reply:
the bot banned me from my dev server and i put that there for it to have me admin so i could use admin commands